site/hxapi/viewamendcancel/amend_simple.md

```diff
@@ -0,0 +1,190 @@
+ | ----        | ----    | --------- | -------- | ---------------------- |
+ | ABTANumber  | String  | [A-Z0-9] 5 chars | Y | This is also known as an 'agent code'. <br>This will be confirmed to you by your Account Manager during set up.|
+ | Password    | String  | [A-Z0-9] 5 chars | N*       | Password required for retail agent requests - intermediaries do not require a password.<br>This will be confirmed to you by your Account Manager during set up.|
+ | Initials    | String  | [A-Z] 3 chars | N  | The initials of the Operator / Agent. |
+ | key         | String  | [A-Z]                                  | Y        | This will be assigned to you by your Account Manager during set up.|
+ | token       | String  | [0-9] 9 chars                         | Y        | This is the same token used in the availability request. |
+ | Email | String | [0-9] 4 chars | Y        | Email address used to make the booking (i.e. the email address of the lead passenger)|
+| System      | String  | [A-Z] 3 chars | Y*       | For European products, you need to pass in the value of `System=ABG` (the default is `System=ABC`, which is UK products only). |
+
+In order to amend the booking with the new details, you only need to pass in the specific parameters you want to amend. For a simple amend these are as follows:
+
+| Name        | Data Type    | Format | Mandatory? | Additional Information |
+| ----        | ----    | --------- | -------- | ---------------------- |
+| Title | String | [0-9] 4 chars | Y        | Title of lead passenger|
+| Initial | String | [A-Z] 1 chars | Y        | Initial of lead passenger|
+| Surname | String | [0-9] 20 chars | Y        | Surname of lead passenger|
+| Address[] | String | [0-9] 20 chars | Y        | First line of address (house name / number and road) of lead passenger. NB: This field can be set to NA|
+| Town | String | [0-9] 4 chars | Y        | Town of address. NB: This field can be set to NA|
+| County | String | [0-9] 4 chars | Y        | County of address. NB: This field can be set to NA|
+| PostCode | String | [0-9] 4 chars | Y        | Post code of address. NB: This field can be set to NA|
+| Email | String | [0-9] 4 chars | Y        | The email address of customer. |
+
+#### For parking bookings:
+
+You may change any of the car details, as indicated in the [car park availability response.](/hxapi/parking/av/carpark)
+
+
+## Amend Booking Response
+
+The amend booking response will return the booking record with all of the updated details.
+
+For a detailed explanation of the fields returned, please see the respective booking requests for [hotels](/hxapi/hotel/bkg), [parking](/hxapi/parking/bkg) and [lounges](/hxapi/lounge/bkg).
+
+## Worked Examples
+
+Below are worked examples of both the request and response for amending (simple) a booking.
+
+### UK Products Amend Booking Request (simple for a car park)
+
+The original booking was made with test details of Mr T TEST:
```

Comment:
Not a big deal, but it says `Mr T TEST` here, but the examples below show `<Initial>A</Initial>` and `<Surname>TESTA</Surname>`.

Reply:
Ah, nevermind, it's the amend... 😊